src/signaler.cpp

```diff
@@ -384,6 +385,18 @@ void zmq::signaler_t::forked ()
 }
 #endif
 
+#if defined ZMQ_HAVE_WINDOWS
```

Comment:
won't the ifdef outside the function break the build on non-windows?

Reply:
The function is only called within a Windows-specific block below. But I will check the travis builds again.

Reply:
Ah right, sorry missed that. Should be fine then.